src/scaffoldmaker/utils/ellipsoidmesh.py

```diff
@@ -314,41 +220,85 @@ def _build_ellipsoid_octant(self, half_counts, axis2_x_rotation_radians, axis3_x
                 start_weight, end_weight, overweighting, end_transition))
         move_x_to_ellipsoid_surface = lambda x: moveCoordinatesToEllipsoidSurface(self._a, self._b, self._c, x)
         move_d_to_ellipsoid_surface = lambda x, d: moveDerivativeToEllipsoidSurface(self._a, self._b, self._c, x, d)
+        def evaluate_surface_d3_ellipsoid_plane(tx, td1, td2):
+            """
+            Restrict d3 to be ellipsoid normal constrained be in plane interpolated from axis_d2 to axis_d3 at tx
```

Comment:
Seems to be missing a "to"- "Restrict d3 to be ellipsoid normal constrained **to** be in plane interpolated...."